src/api.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+
```

Comment:
You could use  **axios.create()** too so you will have a base URL which can be used for all your request functions.